lldb/source/Core/Progress.cpp

```diff
@@ -32,8 +32,8 @@ Progress::~Progress() {
   std::lock_guard<std::mutex> guard(m_mutex);
   if (!m_completed) {
     m_completed = m_total;
-    ReportProgress();
   }
```

Comment:
Remove the braces: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements